data/fixtures/settings.yml

```diff
@@ -3,7 +3,7 @@ QubitSetting:
     name: version
     editable: 0
     deleteable: 0
-    value: 149
+    value: 150
```

Comment:
Ah, I didn't notice that the bumped number was less than my migration. Will fix. Thanks!